src/app/system/codes/view-code/view-code.component.scss

```
@@ -0,0 +1,18 @@
+@media (min-width: 992px) {
+
+  .active-wrapper {
+      position: relative;
+  
+      .active {
+        padding: 0 0 17.5px 0;
+        position: absolute;
+        bottom: 0;
+        right: 0;
+      }
+  }
+
+}
+
+  .options-wrapper {
```

```suggestion
.options-wrapper {
```
Remove any extra space.